docs/08-cicd.md

```diff
@@ -1,67 +1,93 @@
-First you need to get a ["Personal API token"](https://circleci.com/docs/2.0/managing-api-tokens/).
-Then you can trigger a build using the tool `yarn trigger-pipeline`. If you execute it without parameters you will see the usage.
+Steps executed if *Checks* is checked :heavy_check_mark::
+* Linting
+* Flow type checking
+* Unit testing with jest
+* Building the app
 
-
-# Fastlane
+## Fastlane
 
 Fastlane is a task-runner for triggering build relevant tasks. It offers integration with XCode and the Android SDK for building and delivering the app.
 
-## Setup of Fastlane
+### Fastlane Setup
 
-* Install [Ruby >= 2.6.5](https://www.ruby-lang.org/en/documentation/installation/) (The preferred and tested way is to use [Ruby Version Manager (RVM)](https://rvm.io/)).
-  * If using RVM you have to do: `rvm use ruby-2.6.5`
-* Make sure `ruby --version` reports the correct version
-* Run `bundle install --path vendor/bundle` in the project root, in `./android/` and in `./ios/`
-* Run `bundle exec fastlane --version`
+* Install [Ruby >= 2.6.5](https://www.ruby-lang.org/en/documentation/installation/)
+  * The preferred and tested way is to use the [Ruby Version Manager (RVM)](https://rvm.io/).
+  * If using RVM you have to run: `rvm use ruby-2.6.5`.
+* Make sure `ruby --version` reports the correct version.
+* Run `bundle install --path vendor/bundle` in the project root **AND** in `./android/` **AND** in `./ios/`.
+* Run `bundle exec fastlane --version`.
 
-*Hint: You can use `export FASTLANE_SKIP_UPDATE_CHECK=true` to skip the changelog output*
+*Hint: You can run `export FASTLANE_SKIP_UPDATE_CHECK=true` to skip the changelog output.*
 
-## Lanes
+### Lanes
 
 Lanes for Android live in `./android/fastlane` and for iOS in `./ios/fastlane`. Shared lanes are in `./fastlane`.
 
 An overview about FL lanes is available in several documents:
-* [General](../fastlane/README.md#available-actions) - Responsible for delivering and uploading artifacts, bumping the version using the [deliverino GitHub bot](#deliverino-github) and sending notifications.
+* [General](../fastlane/README.md#available-actions) - Responsible for delivering and uploading artifacts.
 * [Android](../android/fastlane/README.md#available-actions) - Responsible for setting up the signing keys and building the Android app.
 * [iOS](../ios/fastlane/README.md#available-actions) - Responsible for setting up the certificates and building the iOS app.
 
-# Apple Certificates and Android Keystore
+## Apple Certificates and Android Keystore
 
 Fastlane is used to setup certificates and keystores. The detailed steps of the CI/CD pipeline are the same as those when manually building the app.
 Therefore, you can follow the documentation for Manual Builds to set up [certificates for iOS](06-manual-builds.md#manually-building-for-ios) and [keystores for Android](06-manual-builds.md#manually-building-for-android).
```

Comment:
Adjust the links and possibly rewrite sections which reference `06-manual-builds.md`